.github/release.yaml

```diff
@@ -28,4 +28,7 @@ changelog:
         - devops
     - title: Other Changes 🧑‍💻
       labels:
-        - "*"
\ No newline at end of file
+        - "*"
+    - title: Dependency Upgrades 🆙
```

Comment:
Do you know if the order matters? If it checks them in order it might just put dependency updates under other changes, but I haven't read any docs or done testing myself.